### ISSUE TITLE
Is DANN valid for semi-supervision?

### ISSUE THREAD

**Issue by WEIZHIHONG720** [xuyu0010/ARID_v1#2]
Hi! Does the DANN approach really work when the training set uses normal illumination data and the test set uses the ARID dataset?

**Comment by xuyu0010** [comment]
Hi, this repository does not contain code for semi-supervised learning.
